.github/workflows/pr-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: PR Tests
+
+on:
+  pull_request:
+
+jobs:
+  tests:
+    name: Tests
+    uses: ./.github/workflows/tests.yml
+    with:
+      python-version: "3.6"
```

Comment:
I expect this will fail for the same reason it fails in the nightly tests.  The shared `tests` workflow runs on ubuntu:latest, and python 3.6 is not available for it.